CollAction/Services/ProjectEmailOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace CollAction.Services
+{
+    public class ProjectEmailOptions
+    {
+        public int MaxNumberProjectEmails { get; set; }
+        public TimeSpan MaxTimeProjectEmails { get; set; }
```

Comment:
After looking it up, I think this means how long the owner can send e-mails after the project ended. Maybe a more descriptive name; TimeAllowedAfterProjectEnds?

Reply:
Changed it